sparkmagic/example_config.json

```diff
@@ -65,5 +65,6 @@
   "livy_server_heartbeat_timeout_seconds": 0,
   "heartbeat_retry_seconds": 10,
 
-  "server_extension_default_kernel_name": "pysparkkernel"
+  "server_extension_default_kernel_name": "pysparkkernel",
+  "authentication_cookie_name": "hadoop.auth"
```

Comment:
Yes this is livy conf. In case of Ldap authentication, livy is setting hadoop.auth cookie and sending in response.